### PR TITLE
Add @hackygolucky as CommComm Director

### PR DESCRIPTION
**Pull request by bnb**
Updates the role of Tracy Hinds as CommComm Director. Not exaclty sure when this land, but it's important to have once we're ready 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The Community Committee is an autonomous committee that collaborates alongside t
 * [bnb](https://github.com/bnb) - **Tierney Cyren** &lt;hello@bnb.im&gt; - **Community Committee Chair**
 * [chowdhurian](https://github.com/chowdhurian) - **Manil Chowdhury** &lt;manil.chowdhury@gmail.com&gt;
 * [dshaw](https://github.com/dshaw) - **Dan Shaw** &lt;dshaw@dshaw.com&gt;
-* [hackygolucky](https://github.com/hackygolucky) - **Tracy Hinds** &lt;tracyhinds@gmail.com&gt;
+* [hackygolucky](https://github.com/hackygolucky) - **Tracy Hinds** &lt;tracyhinds@gmail.com&gt; **Community Committee Director**
```
Comment (vsemozhetbyt) [inline README.md]: A nit: in some cases, we have dashes before statuses, in some we have not. Maybe it is worth to unify?

Comment (bnb) [reply README.md]: Happy to do that in another PR 👍

**Comment by hackygolucky** [comment]
💯

**Comment by bnb** [comment]
Since it's been over a week since the Q&A went up, I'm going to merge this. Congrats @hackygolucky 🎉

**Comment by williamkapke** [comment]
I'm excited to get/have @hackygolucky on board (no pun intended)- but merging this was premature unfortunately. CommComm does not have a charter that supports this role yet and the Foundation's ByLaws still include @ashleygwilliams's position.

**Pull request merged**